gabrielaEnache/src/main/java/collections/TASK.md

```diff
@@ -0,0 +1,80 @@
+Imagine this a s an online retailer where users can:
```

Comment:
It's a good practice to have the task on the hand. However, if you decide to keep it, please, make sure that the name is consistent and suggestive.

Reply:
File name

Reply:
Changed file name to TaskDescription.md